src/App.jsx

```diff
@@ -48,6 +49,7 @@ const App = () => {
                     <Profile />
                   </ProtectedRoute>
                 } />
+              <Route path="/students" element={<Students />}/>
```

Comment:
Shouldn't this be in a ProtectedRoute to prevent accessing it unless you're logged in?